### DISCUSSION TITLE
UserAgent missmatch with Js userAgent

### DISCUSSION THREAD

**Discussion by bbday** [cefsharp/CefSharp#3624] [General]
I use windows 10 x64, default setting, no user agent edit.

Request header show:
Mozilla/5.0 (Windows NT 6.2; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36

I would like it show same value, according by real os in this case windows 10, but if i'll run on different window should be show also right system.

**Comment by amaitland** [comment]


**Reply by amaitland** [reply]
Make sure your application has an app manifest with the relevant compatibility entries. See https://github.com/cefsharp/CefSharp.MinimalExample/blob/master/CefSharp.MinimalExample.WinForms/app.manifest#L43 for example. 

Read https://docs.microsoft.com/en-us/windows/win32/win7appqual/compatibility---application-manifest#leveraging-feature-capabilities for details
